WooCommerce/Classes/ViewRelated/Orders/Order Details/Payment Section/Refunds/IssueRefundTableViewCell.swift

```diff
@@ -0,0 +1,42 @@
+import UIKit
+
+/// Displays a  button to issue new refunds.
+///
+final class IssueRefundTableViewCell: UITableViewCell {
+    @IBOutlet public var issueRefundButton: UIButton!
```

Comment:
nit: this IBOutlet could probably be private?
```suggestion
    @IBOutlet private var issueRefundButton: UIButton!
```